cmd/endpoint.go

```diff
@@ -330,12 +330,12 @@ func CreateEndpoints(serverAddr string, args ...[]string) (string, EndpointList,
 				host = endpoint.Host
 			}
 			hostIPSet, _ := getHostIP4(host)
-			if IPSet, ok := pathIPMap[endpoint.Path]; ok {
-				if !IPSet.Intersection(hostIPSet).IsEmpty() {
+			if ipSet, ok := pathIPMap[endpoint.Path]; ok {
```

Comment:
Keep cleanups to separate PR, this doesn't fix anything related to `if-else` or cyclomatic complexity.

Reply:
ok